apps/files/src/components/FilesAppBar.vue

```diff
@@ -200,6 +200,9 @@ export default {
         }
       }
       return breadcrumbs
+    },
+    freeSpace () {
```

Comment:
I'd have called this "hasFreeSpace" because the current name made think that it's the free space value, not the condition.

Reply:
You're right, fixed